src/styles/bootstrap.scss

```diff
@@ -16,4 +16,61 @@
   @import "bootstrap/scss/helpers";
   @import "bootstrap/scss/buttons";
   @import "bootstrap/scss/forms";
+  @import "bootstrap/scss/utilities";
+  @import "bootstrap/scss/utilities/api";
+
+  // The following are copied from bootstrap.css where the definitions under
+  // :root are otherwise not applied when namespaced so manually defined here
+  --bs-blue: #0d6efd;
```

Comment:
Where are all of these used?

Reply:
They are used indirectly by a lot of the Bootstrap utility classes. The `text-danger` I used in styling the error message is what triggered my including this, but I went ahead and put all of the related definitions in here so we hopefully don't have to try and find / diagnose these in the future when things don't work as expected.